Source/Core/ScreenSpaceEventHandler.js

```diff
@@ -768,5 +776,12 @@ define([
         return destroyObject(this);
     };
 
+    /**
+     * The amount of time, in milliseconds, that mouse events will be disabled after
+     * receiving any touch events, such that any emulated mouse events will be ignored.
+     * @default 800
+     */
+    ScreenSpaceEventHandler.MouseEmulationIgnoreMilliseconds = 800;
```

Comment:
Apparently we are wildly inconsistent here because other objects, like `GeoJsonDataSorce` have static properties the way I described them.  I think Camera is in the minority.  @shunter any preference here?

We should pick one and do it here and then write an issue to update the rest for consistency.  Not sure why we would do all caps, it's weird.